.gitignore

```diff
@@ -28,4 +28,5 @@ yarn-error.log*
 package-lock.json
 yarn.lock
 
-config.json
\ No newline at end of file
+config.json
+.eslintrc.json
```

Comment:
.eslintrc.json is required

Reply:
I had to change line 30 in the .eslintrc.json file because I got this error when I updated my repository:

error  Expected linebreaks to be 'LF' but found 'CRLF'"  linebreak-style

added it to the .ignore file thinking you guys would not get the change since you may not need it.